nero_view.py

```diff
@@ -0,0 +1,211 @@
+import customtkinter
+from tkinter import Frame
+from typing import Optional
+
+customtkinter.set_appearance_mode("dark")
+customtkinter.set_default_color_theme("./themes/ner.json")
+
+
+class NeroView(customtkinter.CTk):
+    def __init__(self, controller) -> None:
+        super().__init__()
+
+        self.controller = controller
+
+        # configure window
+        self.title("NERO")
+        self.geometry(f"{1100}x{580}")
+
+        # configure the grid
+        self.grid_rowconfigure(1, weight=1)
+        self.grid_columnconfigure(0, weight=1)
+
+        # create top and bottom frames
+        self.top_frame = Frame(width=1100, height=300)
+        self.bottom_frame = Frame(width=1100, height=300)
+
+        self.top_frame.grid(row=0, column=0)
+        self.bottom_frame.grid(row=1, column=0)
+
+        # Configure grids for top and bottom frames
+        self.top_frame.grid_rowconfigure(0, weight=1)
+        self.top_frame.grid_columnconfigure(1, weight=1)
+
+        self.bottom_frame.grid_rowconfigure(0, weight=1)
+        self.bottom_frame.grid_columnconfigure(1, weight=1)
+
+        # create the top two frames
+        self.top_right_frame = Frame(self.top_frame, width=550, height=300, bg="black",
+                                     highlightbackground="gray", highlightthickness=1)
+        self.top_left_frame = Frame(self.top_frame, width=550, height=300, bg="black",
+                                    highlightbackground="gray", highlightthickness=1)
+
+        self.top_right_frame.grid(row=0, column=1)
+        self.top_left_frame.grid(row=0, column=0)
+
+        self.top_left_frame.grid_propagate(False)
+        self.top_left_frame.grid_rowconfigure(1, weight=1)
+        self.top_left_frame.grid_columnconfigure(0, weight=1)
+
+        self.top_right_frame.grid_propagate(False)
+        self.top_right_frame.grid_rowconfigure(1, weight=1)
+        self.top_right_frame.grid_columnconfigure(0, weight=1)
+
+        # create the bottom three frames
```

Comment:
Like you said I think we should 100% make each window its own widget/object that inherits from a base widget/class with all the correct formatting